<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Base64Converter</class>
  <widget class="QMainWindow" name="Base64Converter">
    <property name="minimumSize">
      <size>
        <width>800</width>
        <height>600</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Base64Converter</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="4">
          <widget class="QStackedWidget" name="stackedWidget">
            <widget class="QWidget" name="page_TextView">
              <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                  <number>0</number>
                </property>
                <property name="topMargin">
                  <number>0</number>
                </property>
                <property name="rightMargin">
                  <number>0</number>
                </property>
                <property name="bottomMargin">
                  <number>0</number>
                </property>
                <item>
                  <widget class="QPlainTextEdit" name="plainTextEdit" />
                </item>
              </layout>
            </widget>
            <widget class="QWidget" name="page_ImageView">
              <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                  <number>0</number>
                </property>
                <property name="topMargin">
                  <number>0</number>
                </property>
                <property name="rightMargin">
                  <number>0</number>
                </property>
                <property name="bottomMargin">
                  <number>0</number>
                </property>
                <item>
                  <widget class="QLabel" name="label">
                    <property name="text">
                      <string />
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </widget>
        </item>
        <item row="0" column="1">
          <spacer name="verticalSpacer">
            <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>0</width>
                <height>0</height>
              </size>
            </property>
          </spacer>
        </item>
        <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_ConvertToBase64">
            <property name="text">
              <string>➡️</string>
            </property>
          </widget>
        </item>
        <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_ConvertFromBase64">
            <property name="text">
              <string>⬅️</string>
            </property>
          </widget>
        </item>
        <item row="3" column="1">
          <spacer name="verticalSpacer_2">
            <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>0</width>
                <height>209</height>
              </size>
            </property>
          </spacer>
        </item>
        <item row="0" column="2" rowspan="4">
          <widget class="QPlainTextEdit" name="plainTextEdit_Base64" />
        </item>
      </layout>
    </widget>
    <widget class="QMenuBar" name="menubar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>33</height>
        </rect>
      </property>
      <widget class="QMenu" name="menu">
        <property name="title">
          <string>文件</string>
        </property>
        <addaction name="action_Open" />
      </widget>
      <addaction name="menu" />
    </widget>
    <widget class="QStatusBar" name="statusbar" />
    <action name="action_Open">
      <property name="text">
        <string>打开</string>
      </property>
    </action>
  </widget>
  <resources />
  <connections />
</ui>
